src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/ManageApprenticesController.cs

```diff
@@ -23,20 +24,24 @@ public ManageApprenticesController(ICommitmentsService commitmentsService, ICrea
             _createCsvService = createCsvService;
         }
 
-        public async Task<IActionResult> Index(uint providerId)
+        public async Task<IActionResult> Index(uint providerId, string sortField = "", bool reverseSort = false)
         {
             if (!ModelState.IsValid)
             {
                 return BadRequest(ModelState);
             }
 
-            var getApprenticeshipsResponse = await _commitmentsService.GetApprenticeships(providerId);
-            var model = new ManageApprenticesViewModel
-            {
-                ProviderId = providerId,
-                Apprenticeships = getApprenticeshipsResponse?.Apprenticeships
-            };
+            var getApprenticeshipsResponse = await _commitmentsService.GetApprenticeships(providerId, sortField, reverseSort);
+            var model = new ManageApprenticesViewModel{ProviderId = providerId, SortField = sortField, ReverseSort = reverseSort};
```

Comment:
I think this is addressed in another branch but this needs to be done in a mapper instead, to be consistent with the rest of the codebase